tsptw_data_dt.h

```diff
@@ -1101,7 +1118,7 @@ void TSPTWDataDT::LoadInstance(const std::string& filename) {
     else if (relation.type() == "minimum_duration_lapse")
       relType = MinimumDurationLapse;
     else
-      throw "Unknown relation type";
+      throw std::invalid_argument("Unknown relation type");
```

Comment:
Thanks for the fix, it looks like it was me who introduced this `throw "string"` logic in the first place years ago :disappointed_relieved: